templates/user/dashboard/milestones.tmpl

```diff
@@ -27,7 +27,7 @@
 									{{- end -}}
 								]&sort={{$.SortType}}&state={{$.State}}&q={{$.Keyword}}" title="{{.FullName}}">
 								<span class="text truncate">{{$Repo.FullName}}</span>
-								<div class="ui {{if $.IsShowClosed}}red{{else}}green{{end}} label">{{index $.Counts $Repo.ID}}</div>
+								<div class="ui{{if $.IsShowClosed}} red{{else}} green{{end}} label">{{index $.Counts $Repo.ID}}</div>
```

Comment:
In such if+else cases, it's more efficient to keep the space before.

Reply:
One is to reduce one character on the development side and one is to reduce one character on the user side. I prefer to reduce one character on the user side, but it's not a big deal, it's up to you

Reply:
The rendered outcome is the same because one of the if branches will be hit in an if-else. Not a big deal but I guess I prefer the 1 byte reduction.